src/gui/textsel.c

```diff
@@ -281,6 +280,9 @@ void GUI_TextSelectEnablePress(guiTextSelect_t *select, u8 enable)
         }
         return;
     }
+
+#if LCD_DEPTH != 1
+    guiObject_t *obj = (guiObject_t *)select;
```

Comment:
i prefer not to add '#if' unless absolutely necessary.  better to define 'select' at the top of the function and wrap this in a if(LCD_DEPTH != 1) {}

Reply:
sure.